ci/Dockerfile.fedora

```diff
@@ -44,17 +44,17 @@ ARG CXX=g++
 ARG CC=gcc
 ARG BUILD_TYPE=Debug
 ARG CHECK_STYLE=""
+ARG SANITIZE_ADDRESS=OFF
```

Comment:
This is slightly confusing in that some of the options are yes/no and some are ON/OFF. Can we use a single convention everywhere, or do we not have a choice due to compiler flag requirements?

Reply:
To be clear, all the sanitizer options are ON/OFF; I think some options in the past (coverage?) were yes/no.

Reply:
cmake options are conventionally ON/OFF, but cmake seems to support "", FALSE, OFF, and NO as alternatives for not true.  Let's use yes/no everywhere.